java/kotlin-extractor2/src/main/kotlin/entities/Types.kt

```diff
@@ -9,8 +9,13 @@ private fun KotlinUsesExtractor.useClassType(
     c: KaClassType
 ): TypeResults {
     // TODO: this cast is unsafe; .symbol is actually a KaClassLikeSymbol
-    val javaResult = TypeResult(addClassLabel(c.symbol as KaClassSymbol) /* , TODO, TODO */)
-    val kotlinResult = TypeResult(fakeKotlinType() /* , "TODO", "TODO" */)
+    val classId = addClassLabel(c.symbol as KaClassSymbol)
+    val javaResult = TypeResult(classId /* , TODO, TODO */)
+    val kotlinTypeId =
```

Comment:
Is this correct? Shouldn't we check the nullability of the `KaClassType`?

Reply:
Ah, the nullability belongs in `useType`, but it looks like that I got so focused on working out what the dbscheme tables should look like that I didn't add it. I'll add a commit for that.